src/pybind/mgr/dashboard/controllers/rbd_mirroring.py

```diff
@@ -300,7 +311,8 @@ def _get_pool_datum(pool_name):
         data['mirror_images'] = sorted([
             dict({
                 'name': image['name'],
-                'description': image['description']
+                'description': image['description'],
+                'mirror_mode': _get_mirror_mode(ioctx, image['name'])
```

Comment:
Can you add some unit test for this. I see there's `RbdMirroringSummaryControllerTest` so probably a test for checking `mirror_mode` field there would be good to have.